source/extensions/filters/udp/udp_proxy/hash_policy_impl.cc

```diff
@@ -20,13 +20,34 @@ class SourceIpHashMethod : public HashPolicyImpl::HashMethod {
   }
 };
 
+class KeyHashMethod : public HashPolicyImpl::HashMethod {
+public:
+  KeyHashMethod(const std::string& key) : Key(key) {}
+  const std::string& Key;
```

Comment:
Move this member to the private section and rename it to `key_` to conform the coding style.

Reply:
Done it! Thanks.